src/components/KnownDrugs/FilterTable.js

```diff
@@ -102,194 +176,203 @@ class KnownDrugsDetail extends React.Component {
     return (
       <React.Fragment>
         <div className={classes.dcChartContainer}>
-          <DCContainer id="dc-trial-by-status-chart" title="Trials by Status" />
-          <DCContainer id="dc-trial-by-phase-chart" title="Trials by Phase" />
-          <DCContainer id="dc-trial-by-drug-chart" title="Trials by Drug" />
-          <DCContainer
-            id="dc-drug-by-activity-chart"
-            title="Drug by Activity"
-          />
-          <DCContainer
-            id="dc-drug-and-disease-by-activity-chart"
-            title="Activity by (Disease, Drug)"
-          />
+          <div className={classes.dcChartSection}>
+            <strong>Summary</strong>
+            <p>
+              <span
+                id="unique-drugs-count"
+                className={classNames(
+                  classes.countLabel,
+                  classes.countLabelDrug
+                )}
+              />{' '}
+              unique drugs
+            </p>
+            <p>
+              <span
+                id="associated-targets-count"
+                className={classNames(
+                  classes.countLabel,
+                  classes.countLabelTarget
+                )}
+              />{' '}
+              associated targets
+            </p>
+            <p>
+              <span
+                id="associated-diseases-count"
+                className={classNames(
+                  classes.countLabel,
+                  classes.countLabelDisease
+                )}
+              />{' '}
+              associated diseases
+            </p>
+            <div className="clearfix" />
+          </div>
+
+          <div className={classes.dcChartSection}>
+            <DCContainer
+              id="dc-trial-by-phase-chart"
+              title="Clinical trials by phase"
+            />
+          </div>
+          <div className={classes.dcChartSection}>
+            <DCContainer
+              id="dc-drug-by-type-chart"
+              title="Unique drugs by type"
+            />
+          </div>
+          <div className={classes.dcChartSection}>
+            <DCContainer
+              id="dc-drug-by-activity-chart"
+              title="Unique drugs by activity"
+            />
+          </div>
         </div>
         <OtTable
           loading={false}
           error={null}
           columns={columns}
           data={filteredRows}
+          tableLayout="fixed"
         />
       </React.Fragment>
     );
   }
+
   renderCharts = () => {
     const { rows } = this.props;
 
     // connect
     const ndx = crossfilter(rows);
 
     // dimensions
-    const dimStatus = ndx.dimension(d => d.status || 'UNKNOWN');
-    const dimPhase = ndx.dimension(d => d.phase);
-    const dimActivity = ndx.dimension(d => d.activity);
-    const dimDrugAndDisease = ndx.dimension(d => [d.drugName, d.efoLabel]);
-    const dimDrug = ndx.dimension(d => d.drugName);
+    const dimPhase = ndx.dimension(d => d.clinicalTrial.phase);
```

Comment:
I think we could stop using `ndx` as the variable name that holds the crossfilter instance. In the DC.js example, they use that variable name because they are dealing with Nasdaq index data. We could use a more descriptive name here. What do you think?

Reply:
Thanks Alfredo, yes I agree.

Reply:
I've pushed the change. I've opted for the 'xf' suffix for 'crossfilter'.